tests/tests_test.go

```diff
@@ -11,6 +11,12 @@ import (
 )
 
 func TestFunctional(t *testing.T) {
+
+	host := os.Getenv("BRIGADE_BRIGADE_GITHUB_GW_SERVICE_HOST")
```

Comment:
So, we can set this to e.g. `brigade-brigade-github-gw` to actually run the test against the brigade cluster? (Great!)